### PR TITLE
Results page

### PR DESCRIPTION
**Pull request by gregorsart**

### CODE REVIEW

File: components/DogCard.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+`;
+
+const StyledParagraph = styled.p`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 1rem;
+  padding: 0 1rem;
+`;
```
Comment (doemser) [inline components/DogCard.js:74]: Here you use a lot of styling I would expect on container elements, but not necessarily on a paragraph element. On a p tag you could achieve centering of the text by setting` text-align: center;` for example. 

Make sure you are getting the semantically best out of it, a StyledPoint seems to be more of a div, then a p tag for me

File: components/QuizForm.js [hunk 2]
```diff
@@ -238,12 +246,14 @@ const StyledInput = styled.input`
...
-  display: table-cell;
-  vertical-align: middle;
-  padding: 0.7rem 1em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
   text-align: center;
+  /* display: table-cell;
+  vertical-align: middle; */
```
Comment (doemser) [inline components/QuizForm.js:255]: remove comments

File: pages/api/quizResults/index.js [hunk 3]
```diff
@@ -1,37 +1,56 @@
...
+    //         `https://api.api-ninjas.com/v1/dogs?barking=${barking}&offset=${offset}`,
+    //         {
+    //           headers: {
+    //             "X-Api-Key": process.env.DOG_API_KEY,
+    //           },
+    //         }
+    //       )
+    //     );
+    //   }
+    // }
```
Comment (doemser) [inline pages/api/quizResults/index.js:41]: remove unused code

File: pages/quiz-results/index.js [hunk 4]
```diff
@@ -1,29 +1,34 @@
...
+import useSWR from "swr";
 
-export default function QuizResults({ assignedPoints }) {
-  if (assignedPoints.length === 0) {
-    return null;
-  }
+import { useRouter } from "next/router";
+import DogCard from "@/components/DogCard";
+
+const fetcher = (url) => fetch(url).then((response) => response.json());
```
Comment (doemser) [inline pages/quiz-results/index.js:8]: you can provide this fetcher function globally, how to do this is described here:

https://swr.vercel.app/docs/global-configuration

File: styles.js [hunk 5]
```diff
@@ -5,10 +5,25 @@ export default createGlobalStyle`
...
+    display: block;
+    max-width: 100%;
+  }
+
+  input, button, textarea, select {
+    font: inherit;
+  }
+
+  p, h1, h2, h3, h4, h5, h6 {
+    overflow-wrap: break-word;
```
Comment (doemser) [inline styles.js:22]: interesting global styles. how did you come up with it?

File: styles.js [hunk 6]
```diff
@@ -31,38 +46,17 @@ export default createGlobalStyle`
   }
 
   fieldset {
-    border: 2px solid var(--soft-background);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    border: 0;
```
Comment (doemser) [inline styles.js:53]: why you prefer this over having a styled-component?

File: styles.js [hunk 7]
```diff
@@ -5,10 +5,25 @@ export default createGlobalStyle`
   *::before,
```
Comment (doemser) [inline styles.js:5]: here you should be using createGlobalStyles from styled-components

File: styles.js (hunk 5 above)
Comment (gregorsart) [reply styles.js:22]: we got it from here: [css reset](https://www.joshwcomeau.com/css/custom-css-reset/)

File: styles.js (hunk 6 above)
Comment (gregorsart) [reply styles.js:53]: we often tested things first in the `global styles` and then we implemented it as styled components. In this case we forgot to implement.

File: pages/quiz-results/index.js (hunk 4 above)
Comment (gregorsart) [reply pages/quiz-results/index.js:8]: ty, good idea.

File: pages/quiz-results/index.js [hunk 8]
```diff
@@ -5,15 +5,14 @@ import useSWR from "swr";
 import { useRouter } from "next/router";
 import DogCard from "@/components/DogCard";
 
-const fetcher = (url) => fetch(url).then((response) => response.json());
+// const fetcher = (url) => fetch(url).then((response) => response.json());
```
Comment (NadjaKanunnikow) [inline pages/quiz-results/index.js:8]: you can delete this line.

File: pages/_app.js [hunk 9]
```diff
@@ -1,13 +1,22 @@
...
-      <GlobalStyle />
-      <Layout>
-        <Component {...pageProps} />
-      </Layout>
+      <GlobalStyle />{" "}
+      <SWRConfig
+        value={{
+          refreshInterval: 3000,
+          fetcher: (resource, init) =>
+            fetch(resource, init).then((res) => res.json()),
```
Comment (NadjaKanunnikow) [inline pages/_app.js:13]: below you wrote response instead of res. you can do that here aswell to make it more consistent.

**Review by NadjaKanunnikow — APPROVED** [review]
looks good!

**Comment by gregorsart** [comment]
@NadjaKanunnikow ty for reviewing!

**Pull request merged**